### PR TITLE
Chore/cleanup

### PR DESCRIPTION
**Pull request by Gawdfrey**
Cleaning up unused logs, components and imports.
Exposes the search term in the provider, as it displayed undefined after searching for a term

### CODE REVIEW

File: src/context/GiphyProvider.tsx [hunk 1]
```diff
@@ -147,6 +145,7 @@ const GiphyProvider = ({
         selectedSearchType,
         apiKey,
         items: data,
+        searchTerm,
```
Comment (XSlemX) [inline src/context/GiphyProvider.tsx:148]: Is this needed? Is it in use?

**Review by XSlemX — APPROVED** [review]
Lgtm

File: src/context/GiphyProvider.tsx (hunk 1 above)
Comment (Gawdfrey) [reply src/context/GiphyProvider.tsx:148]: Yes! It is used in the title above the results and display for example: "Showing results for `Stacc`". Without it the title would display: "Showing results for `Undefined`".

**Pull request merged**